pipelines/managed/release-to-github/README.md

```diff
@@ -20,6 +20,12 @@ Tekton release pipeline to release binaries extracted from the image built with
 | taskGitUrl                      | The url to the git repo where the release-service-catalog tasks to be used are stored                                              | Yes      | https://github.com/konflux-ci/release-service-catalog.git |
 | taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                                                     | No       | -                                                         |
 
+## Changes in 4.7.0
+* Pass taskGitUrl and taskGitRevision to base64-encode-checksum task
+
+## Changes in 4.6.0
```

Comment:
I think this should just be 4.6.0 with 2 entries

Reply:
fixed